tsconfig.json

```diff
@@ -19,11 +20,11 @@
       "dom"
     ],
     "paths": {
-      "entity-state": [
-        "dist/entity-state"
+      "@ngxs-labs/entity-state": [
```

Comment:
Shouldn't these aliased paths only be available within the integration app, and to the dist folder?

Reply:
At least I think that this is how it is set up in the ngxs main repo?

Reply:
Yeah, I moved the paths into the integration config files.